### PR TITLE
Add notes for alphas for Chocolatey components

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for alpha releases of all Chocolatey components

## Motivation and Context

There are releases coming up, and we need to get this documented

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,33 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Bug
+
+- Fix `choco upgrade` and `choco install` commands that were failing when using the `--version` option.
+
+### Enhancements
+
+- Add back support for `--disable-repository-optimizations` options which was missing in previous alpha release.
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:45]:
```suggestion
- Add back support for the `--disable-repository-optimizations` option which was missing in previous alpha release.
```

File: input/en-us/chocolatey-gui/release-notes.md [hunk 2]
```diff
@@ -16,6 +16,28 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
...
+>
+> If you run into any problems when using this alpha version of Chocolatey GUI we would ask that you comment on this [discussion](https://github.com/chocolatey/choco/discussions/2995), which is where we will be collating issues, and providing workarounds, etc.  We will not be accepting issues raised against this alpha release.
+
+### Known Issues
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Bug
+
+- Chocolatey GUI MSI incorrectly included the chocolatey.dll and chocolatey.pdb files - see [#974](https://github.com/chocolatey/ChocolateyGUI/issues/974).
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:39]:
```suggestion
- Chocolatey GUI MSI incorrectly included the `chocolatey.dll` and `chocolatey.pdb` files - see [#974](https://github.com/chocolatey/ChocolateyGUI/issues/974).
```

Comment (pauby) [reply input/en-us/chocolatey-gui/release-notes.md:39]: I'm not sure if we have done this previously but we need to highlight these are files. Thoughts?

File: input/en-us/licensed-extension/release-notes.md [hunk 3]
```diff
@@ -40,6 +40,24 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
...
+>
+> If you run into any problems when using this alpha version of Chocolatey Licensed Extension we would ask that you comment on this [discussion](https://github.com/chocolatey/choco/discussions/2995), which is where we will be collating issues, and providing workarounds, etc.  We will not be accepting issues raised against this alpha release.
+
+### Known Issues
+
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Enhancement
+
+- Improve warning messages regarding dependncy resolution problems during `choco install` and `choco upgrade`
```
Comment (pauby) [inline input/en-us/licensed-extension/release-notes.md:59]:
```suggestion
- Improve warning messages regarding dependency resolution problems during `choco install` and `choco upgrade`
```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/chocolatey-gui/release-notes.md (hunk 2 above)
Comment (gep13) [reply input/en-us/chocolatey-gui/release-notes.md:39]: This makes sense to me, yes.  I will get that updated!

**Review by pauby — DISMISSED** [review]

File: input/en-us/choco/release-notes.md [hunk 4]
```diff
@@ -18,6 +18,34 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+See this [list](https://github.com/chocolatey/choco/discussions/2995) for known issues with this pre-release.
+
+### Bug
+
+- Fix `choco search` command that was providing invalid results when using the `--version` option.
+- Fix dependency resolution problems during execution of `choco upgrade` and `choco install`.
+
+### Enhancements
+
+- Add back support for the `--disable-repository-optimizations` option which was missing in previous alpha release.
```
Comment (corbob) [inline input/en-us/choco/release-notes.md:46]:
```suggestion
- Add back support for the `--disable-repository-optimizations` option which was missing in previous alpha releases.
```

As there have been multiple releases, perhaps indicate it was missing from all of them?

**Review by corbob — COMMENTED** [review]
Just the one note on the tense/grammar of referencing prior alphas.

**Pull request merged**